content/en/docs/concepts/workloads/controllers/deployment.md

```diff
@@ -1027,8 +1027,23 @@ Conditions:
   Progressing   True    NewReplicaSetAvailable
 ```
 
-`type: Available` with `status: "True"` means that your Deployment has minimum availability. Minimum availability is dictated
-by the parameters specified in the deployment strategy. `type: Progressing` with `status: "True"` means that your Deployment
+`type: Available` with `status: "True"` means that your Deployment is healthy and has a minimum availability (`.status.availableReplicas` is at least equal to `.spec.replicas  - maxUnavailable`).
+`maxUnavailable` is computed according to the deployment strategy:
+- For the `RollingUpdate` strategy, it is a resolved value of `.spec.strategy.rollingUpdate.maxUnavailable`.
+- For the `Recreate` strategy, it is always equal to 100% of the replicas that we allow to be down, so that new rollouts can proceed.
```

Comment:
This counts with a change proposed in https://github.com/kubernetes/kubernetes/pull/113228. Currently it is 0.